src/commands/dev-env-import-sql.ts

```diff
@@ -78,15 +88,14 @@ export class DevEnvImportSQLCommand {
 			const expectedDomain = `${ this.slug }.${ lando.config.domain }`;
 			await validateSQL( resolvedPath, {
 				isImport: false,
-				skipChecks: [],
+				skipChecks: isMyDumper ? [ 'dropTable', 'dropDB' ] : [],
```

Comment:
`dropTable` and `dropDB` validations doesn't make sense in the mydumper context because `isMyDumper` uses `--overwrite-table` to perform any table/db dropping - hence we're ignoring it here.